Replaces style attribute with class for spinner

Kristyn-Mimura opened the pull request:
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1427)

# List of squashed commits
- Updates spacing

# Test Checklist

- [X] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [X] General Visual Inspection:

mhodgesatuh reviewed (approved):
Nice cleanup

mhodgesatuh commented:
src/main/resources/static/scss/components/_spinner.scss
Minor icon MINOR
Code Style
Unexpected missing end-of-source newline (no-missing-end-of-source-newline)
}

Looks like one bit of cleanup to take care of.

Kristyn-Mimura commented:
> src/main/resources/static/scss/components/_spinner.scss Minor icon MINOR Code Style Unexpected missing end-of-source newline (no-missing-end-of-source-newline) }
> 
> Looks like one bit of cleanup to take care of.

I added a newline, squashed the commits then pushed to origin but the newline didn't show up, I'm trying to figure out what went wrong

mhodgesatuh commented:
Hopefully your IDE is not configured to remove the end-of-source newline.


On Fri, Apr 7, 2023 at 7:26 PM Kristyn-Mimura ***@***.***>
wrote:

> src/main/resources/static/scss/components/_spinner.scss Minor icon MINOR
> Code Style Unexpected missing end-of-source newline
> (no-missing-end-of-source-newline) }
>
> Looks like one bit of cleanup to take care of.
>
> I added a newline, squashed the commits then pushed to origin but the
> newline didn't show up, I'm trying to figure out what went wrong
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/pull/783#issuecomment-1500796308>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACX2CME5FHODZCJ2EZI5DX3XADZIVANCNFSM6AAAAAAWXGZ4LY>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

JorWo commented:
Maybe try add two newlines at the end of _spinner.scss to fix the codacy issue.

Kristyn-Mimura commented:
> Maybe try add two newlines at the end of _spinner.scss to fix the codacy issue.

Thanks! That worked!

JorWo commented:
A recent merge just created a merge conflict.

JorWo commented on src/main/resources/templates/admin.html:
```diff
@@ -32,8 +32,8 @@ <h1 class="text-center text-md-left">UH Groupings Administration</h1>
             <div class="d-flex justify-content-center">
-                <div class="spinner-border text-light" style="width: 7rem; height: 7rem; margin-top: 17em;" ng-show="loading"></div>
-            </div>
+            <div class="spinner-border text-light spinner-style" ng-show="loading"></div>
+                </div>
```
Could you fix the spacing here? After that, everything in this PR is good to go.

JorWo commented:
Sorry about that, there's another merge conflict.
